indicators/NO_FUNC_001-004/R/NO_FUNC_001-004.qmd

```diff
@@ -0,0 +1,3139 @@
+
+-->
+### Data sets
+<!-- Please print the entire workflow with echo=T. Also, simply reading cached data creates loose ends - I cant find out where this data comes from. You may also chose to read tin this data (or a cached version of it) way at the top of the file, so that you can add some in-line R code for the summary stats in the introduction part (see previous comments)-->
+
+Download ANO testing data from Miljødirektoratet
+```{r, eval = F, echo = F}
+# download 
+url <- "https://nedlasting.miljodirektoratet.no/naturovervaking/naturovervaking_eksport.gdb.zip"
+download(url, dest="P:/41201785_okologisk_tilstand_2022_2023/data/ANO/naturovervaking_eksport.gdb.zip", mode="wb") 
+unzip("P:/41201785_okologisk_tilstand_2022_2023/data/ANO/naturovervaking_eksport.gdb.zip", exdir = "P:/41201785_okologisk_tilstand_2022_2023/data/ANO/naturovervaking_eksport.gdb")
+```
+
+Import testing data:
+```{r}
+#Sys.setlocale("LC_ALL", "no_NB.utf8") #works with æøå or use "Norwegian"
+# ANO data
+ANO.sp <- st_read("P:/41201785_okologisk_tilstand_2022_2023/data/naturovervaking_eksport.gdb",
+                  layer="ANO_Art", quiet = T)
+ANO.geo <- st_read("P:/41201785_okologisk_tilstand_2022_2023/data/naturovervaking_eksport.gdb",
+                   layer="ANO_SurveyPoint", quiet = T)
+#head(ANO.sp)
+#head(ANO.geo)
+
+# GRUK data
+GRUK.variables<-readRDS(paste0(here::here(),"/indicators/NO_FUNC_001-004/data/cache/GRUK.variables.RDS"))
+GRUK.species<-readRDS(paste0(here::here(),"/indicators/NO_FUNC_001-004/data/cache/GRUK.species.RDS"))
+GRUK2021.condition<-readRDS(paste0(here::here(),"/indicators/NO_FUNC_001-004/data/cache/GRUK2021.condition.RDS"))
+
+# ASO data
+ASO.sp<-readRDS(paste0(here::here(),"/indicators/NO_FUNC_001-004/data/cache/ASO.sp.RDS"))
+ASO.geo<-readRDS(paste0(here::here(),"/indicators/NO_FUNC_001-004/data/cache/ASO.geo.RDS"))
+
+# 3Q data
+
+
+```
+
+Import Plant indicators from Tyler et al. (2021):
+```{r}
+## Tyler indicator data
+#ind.Tyler <-readRDS(paste0(here::here(),"/indicators/NO_FUNC_001-004/data/cache/ind.dat.RDS"))
+ind.Tyler <- read.table("P:/41201785_okologisk_tilstand_2022_2023/data/functional plant indicators/Tyler et al_Swedish plant indicators.txt", sep = '\t', header=T, quote = '')
+# #head(ind.Tyler)
+ 
+ind.Grime<-readRDS(paste0(here::here(),"/indicators/NO_FUNC_001-004/data/cache/ind.Grime.RDS"))
+
+
+```
+
+Import reference species lists (Generalized species lists from NiN & from Evju et al. 2023)
+```{r}
+# data from cache
+Eco_State<-readRDS(paste0(here::here(),"/indicators/NO_FUNC_001-004/data/cache/Eco_State.RDS"))
+# load("/data/P-Prosjekter2/41201785_okologisk_tilstand_2022_2023/functional plant indicators/reference from NiN/Eco_State.RData")
+# str(Eco_State)
+natopen_NiN_ref<-readRDS(paste0(here::here(),"/indicators/NO_FUNC_001-004/data/cache/natopen_NiN_ref.RDS"))
+natopen_NiN_ref_spInfo<-readRDS(paste0(here::here(),"/indicators/NO_FUNC_001-004/data/cache/natopen_NiN_ref_spInfo.RDS"))
+```
+
+#### Data handling
```

Comment:
Some more text describing the different steps would be useful.

First we do some data handling. ...